### PR TITLE
Add vr tests to vNext Spinner

### PR DESCRIPTION
**Pull request by tomi-msft**
This PR is to add vr-tests for vNext Spinner

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/67dd475e49182d17dbf7d9195c5df9bce02b7b3c'>67dd475e49182d17dbf7d9195c5df9bce02b7b3c</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/vr-tests-react-components/src/stories/Spinner.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  })
+  .addStory('Inverted with Label', () => <Spinner className="test-class" appearance="inverted" label="Loading" />, {
+    includeHighContrast: true,
+    includeDarkMode: true,
+  })
+  .addStory(
+    'Inverted with Label Before',
+    () => <Spinner className="test-class" appearance="inverted" labelPosition="before" label="Loading" />,
+    {
+      includeHighContrast: true,
```
Comment (spmonahan) [inline apps/vr-tests-react-components/src/stories/Spinner.stories.tsx:88]: Would the `labelPosition="before"` and `labelPosition="after"` stories benefit from including RTL variants?

File: apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx [hunk 2]
```diff
@@ -41,3 +42,46 @@ export const TestWrapperDecoratorFullWidth: DecoratorFunction<ExtendedStoryFnRet
...
+//         padding: '10px',
+//         overflow: 'hidden',
+//         transition: 'none !important',
+//         animation: 'none !important',
+//       }}
+//     >
+//       {story()}
+//     </div>
+//   </div>
+// );
```
Comment (khmakoto) [inline apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx:60]: This comment should be removed.

**Comment by khmakoto** [comment]
@tomi-msft if the screener changes look correct I can go ahead and approve them.

**Comment by tomi-msft** [comment]
@khmakoto Yeah, they look correct to me

File: apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx (hunk 2 above)
Comment (khmakoto) [inline apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx:60]: This comment should be removed

```suggestion
```

File: apps/vr-tests-react-components/src/stories/Spinner.stories.tsx (hunk 1 above)
Comment (tomi-msft) [reply apps/vr-tests-react-components/src/stories/Spinner.stories.tsx:90]: Turns out it looks the same as its inverse. i.e. `labelPosition="before"` looks like `labelPosition="after"` in RTL and vice versa

**Review by khmakoto — APPROVED** [review]

**Pull request merged**